On my Raspberry Pi box armv7, The values needed are u32. However seems for
Alpine 32bit systems, u64 is needed.

diff --git a/src/builtins/ulimit.rs b/src/builtins/ulimit.rs
index 8e7d59b..e99e7a3 100644
--- a/src/builtins/ulimit.rs
+++ b/src/builtins/ulimit.rs
@@ -160,15 +160,9 @@ fn set_limit(limit_name: &str, value: u64, for_hard: bool) -> String {
     }
 
     if for_hard {
-        #[cfg(target_pointer_width = "32")]
-        { rlp.rlim_max = value as u32; }
-        #[cfg(target_pointer_width = "64")]
-        { rlp.rlim_max = value; }
+        rlp.rlim_max = value;
     } else {
-        #[cfg(target_pointer_width = "32")]
-        { rlp.rlim_cur = value as u32; }
-        #[cfg(target_pointer_width = "64")]
-        { rlp.rlim_cur = value; }
+        rlp.rlim_cur = value;
     }
 
     unsafe {
